test/behavior.zig

```diff
@@ -96,6 +96,10 @@ test {
     _ = @import("behavior/void.zig");
     _ = @import("behavior/while.zig");
 
+    if (builtin.zig_backend == .stage2_wasm) {
```

Comment:
I suggest to delete the other import in this file and do this:

```suggestion
    if (builtin.target.cpu.arch == .wasm32) {
```